### ISSUE TITLE
Getting remote server error

### ISSUE THREAD

**Issue by Skipups** [SkiFilmReviews/snow-forecast-sfr#41]
<img width="1406" alt="Screen Shot 2020-06-01 at 10 16 37 PM" src="https://user-images.githubusercontent.com/47464568/83472567-91972a80-a455-11ea-84cd-115c89f469e4.png">

I am trying to use the API in development but getting blocked by CORS. Do you have any recommendations?

**Comment by MikeSpitz** [comment]
Hey, thanks for highlighting.

That is a valid error you are getting. The best way to get around that is to create a proxy API where you do the snowRequest on the backend (that you control). This is the best, most concrete, solution.

But there's a valid usecase here where you just want to play around with the module and not have to create a backend to use it and there are also proxy site services (e.g. https://github.com/Freeboard/thingproxy ) that help you get around this issue. 

I'm going to release a fix for this tonight (UK time) so that you can use one of these third party services without having to create the proxy API. I'll update in here when done.

**Comment by Skipups** [comment]
Thanks so much Mike. I'd be happy to update the readme afterwards. I also think adding including the resort logo image would be great. Would you like me to do that?

**Comment by MikeSpitz** [comment]
This is fixed in https://github.com/SkiFilmReviews/snow-forecast-sfr/pull/42

Make sure your package.json has 2.0.1 for this module. You can see the docs in the readme under Options / CORS

Should stress again that this should be happening on the backend because of how long it can take to fetch the site and then parse it. That way you can add a caching layer a little better too.

With regards to the resort logo, sounds great! Feel free to open a PR for it or you can create another issue and I can see when I can get around to doing it. Hope that's helped you out. Am closing for now but feel free to re-open if somethings not right.
